packages/react-checkbox/src/components/Checkbox/Checkbox.types.ts

```diff
@@ -125,11 +124,6 @@ export interface ICheckboxProps
    */
   checkmarkIconProps?: IIconProps;
 
-  /**
-   * Optional keytip for this checkbox
-   */
-  keytipProps?: IKeytipProps;
```

Comment:
Should this (and other removed props) have been marked as deprecated already in 7?

Reply:
Isn't it too late for that ? :/

Reply:
We could still add it in the 7 branch so there's a bit of warning maybe? Though that assumes they have the deprecation lint rule enabled (and are using things in a way the rule can catch). :(